openapi/src/main/kotlin/com/expediagroup/sdk/generators/openapi/OpenApiSdkGenerator.kt

```diff
@@ -69,15 +71,15 @@ class OpenApiSdkGenerator {
 
     fun run() {
         try {
+            // Adjust namespace to fit with JVM package naming conventions
+            val packageName = namespace.lowercase().replace(NON_ALPHANUMERIC_REGEX, "")
             val config = CodegenConfigurator().apply {
-                // Adjust namespace to fit with JVM package naming conventions
-                val packageName = namespace.lowercase().replace(Regex("[^a-z0-9]"), "")
                 // specify the target language
                 setGeneratorName("kotlin")
                 setTemplateDir("templates/openworld-sdk")
-                setInputSpec(
-                    prepareSpecFile()
-                )
+                val path = prepareSpecFile()
+                val processedPath = preProcessSpecFile(path)
```

Comment:
If we kept this variable, let's go with `processedFilePath` as we process the file and not the path.